api/views/student.py

```diff
@@ -293,6 +294,36 @@ def active_students_with_no_groups(self, request: Request) -> Response:  # noqa:
             status=status.HTTP_200_OK,
         )
 
+    @extend_schema(
+        request=DashboardFinishedOralInterviewSerializer,
+        responses={
+            status.HTTP_204_NO_CONTENT: OpenApiResponse(description="Action is taken"),
+            status.HTTP_409_CONFLICT: OpenApiResponse(
+                response=BaseAPIExceptionSerializer,
+                description="Only with status NEEDS_INTERVIEW_TO_DETERMINE_LEVEL can be processed",
+            ),
+            status.HTTP_422_UNPROCESSABLE_ENTITY: OpenApiResponse(
```

Comment:
I suggest using this status everywhere where we have not found some entity, so use it instead of 409, who do you think?

Reply:
yes, let's go for 422. I searched through old messages in our TG chat and see that @Roxe322 was in favor of 422 over 409 in similar cases. The main argument for 422 is that it means that the client [should not repeat the request without modificatio](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422), which is exactly what we want to tell the client. 409 doesn't necessary entail that.